lib/nexpose.rb

```diff
@@ -57,25 +57,27 @@
 require 'ipaddr'
 require 'json'
 require 'cgi'
+require 'nexpose/api'
+require 'nexpose/json_serializer'
 require 'nexpose/error'
 require 'nexpose/util'
 require 'nexpose/alert'
 require 'nexpose/ajax'
-require 'nexpose/api'
 require 'nexpose/api_request'
 require 'nexpose/asset'
 require 'nexpose/common'
 require 'nexpose/console'
 require 'nexpose/credential'
-require 'nexpose/site_credential'
+require 'nexpose/site_credentials'
 require 'nexpose/shared_credential'
+require 'nexpose/web_credentials'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.